tools/fiptool/tbbr_config.c

```diff
@@ -6,7 +6,7 @@
 
 #include <stddef.h>
 
-#include "firmware_image_package.h"
+#include <firmware_image_package.h>
```

Comment:
Please move these includes to be in alphabetical order, to line 7.

Also, please update the copyright date to 2016-2017.